server/implementation/pom.xml

```diff
@@ -26,6 +26,12 @@
             <artifactId>smallrye-graphql-schema-model</artifactId>
         </dependency>
 
+        <!-- Jandex indexer -->
+        <dependency>
```

Comment:
We shouldn't use Jandex in the runtime module... Unless it's absolutely necessary.
I don't understand why exactly you need this? Can the Coercing classes somehow work with instances of the annotations themselves instead of the Jandex object?

Reply:
I fixed `io.smallrye.graphql.schema.helper.Directives` to avoid converting to `AnnotationInstances`.
ca0fd64b79a8cd71c16fbcc6642ff489439225e8